docs/src/main/asciidoc/security-jwt.adoc

```diff
@@ -1014,7 +1016,7 @@ include::{generated-dir}/config/quarkus-smallrye-jwt.adoc[opts=optional, levelof
 [cols="<m,<m,<2",options="header"]
 |===
 |Property Name|Default|Description
-|mp.jwt.verify.publickey|none|The `mp.jwt.verify.publickey` config property allows the Public Key text itself to be supplied as a string.  The Public Key will be parsed from the supplied string in the order defined in section <<Supported Public Key Formats>>.
+|mp.jwt.verify.publickey|none|The `mp.jwt.verify.publickey` config property allows the Public Key text itself to be supplied as a string.  The Public Key will be parsed from the supplied string in the order defined in section xref:Supported Public Key Formats[].
```

Comment:
Not critical as it works, but while we are here fixing anchors, would be good to do it consistently by linking to the ID [[ ]].

Reply:
Works fine, but will add the anchors